projects/angular/src/forms/common/wrapped-control.spec.ts

```diff
@@ -193,6 +193,46 @@ class WithControlAndSuccess {
   model = '';
 }
 
+@Component({
+  template: `
+    <form-wrapper>
+      <test-wrapper3>
+        <label> Label </label>
+        <textarea testControl3 [formControl]="form.get('control') || form.get('alternative')"></textarea>
+        <clr-control-success>Successful!</clr-control-success>
+      </test-wrapper3>
+      <form-wrapper> </form-wrapper
+    ></form-wrapper>
+  `,
+})
+class WithDynamicFormControl {
+  form = new FormGroup<any>({
+    alternative: new FormControl(),
+  });
+
+  addControl() {
+    this.form.addControl('control', new FormControl('TEST'));
+  }
+}
+@Component({
+  template: `
+    <form-wrapper>
+      <test-wrapper3>
+        <label> Label </label>
+        <textarea testControl3 [(ngModel)]="object['control']"></textarea>
+        <clr-control-success>Successful!</clr-control-success>
+      </test-wrapper3>
+      <form-wrapper> </form-wrapper
+    ></form-wrapper>
+  `,
+})
+class WithDynamicNgControl {
+  object = {};
```

Comment:
object is a reserved keyword. It's better to use another variable name

Reply:
Thanks!